_digest/january-2021/next-on-self-sabotage-time-scarcity.mdx

```diff
@@ -0,0 +1,152 @@
+control over it. Remind yourself that you are not going against time. You have to work with it. 
+A simple way to replace this sentence in our vocabulary is by saying, “This is not my priority” 
+or a simple “no.” Using these phrases brings back the responsibility to you and helps you use your 
+time under your control. </p>
+
+
+<div align="center">
+    <figure>
+        <img src="/blog/january-2021/on-phone-with-clock.png" alt="Lady on phone with clock in background" width="500"/>
+    </figure>
+</div>
+
+
+<h3>For those attending online classes: Be aware of how you are using your time.</h3>
+
+<p> Make sure you manage your time so that you aren’t hooked up to screens 24/7. 
+Your leisure time must be spent away from screens—otherwise, there is no distinction between 
+work and play. It’s the same thing when you are told not to do your math work in bed. The purpose 
+of a bed is to get some rest and sleep on it, not study. </p>
+
+<p> It is all about your environment. Spending your free time online creates the illusion that we don’t 
+have free time because we’re spending it on things we need to do, or should do. 
+When I realized this problem during quarantine, I started using the Pomodoro technique. 
+I’m sure most students know how this works, but in case you don’t, it’s essentially a 
+way to maintain your focus during work. I set my Pomodoro timer for around 25 minutes, and 
+when time’s up, I know that it’s time to take a break. I spend these 5-10 minutes getting water 
+or a snack or taking a short walk around my house to maintain my energy levels. </p>
+
+<p> Ensure you are not using your break time going on social media or reading your emails as 
+this can produce unnecessary feelings of stress and may cause you to work on another task. 
+Then, It’s quite likely that you will get into that loop of feeling like there’s not enough time 
+for the things you want to do, all over again. </p>
+
+<p> When we are immersed in this bubble of spending our break time online, we feel the need to compete. </p>
+
+<h3> Final words 🙂 </h3>
+
+<p> A lot of people struggle with this, so you don’t need to feel alone! 
+The first step to combatting this problem is becoming aware of it, which I just helped you overcome 🙂 
+After the first step, it becomes more comfortable as you come to terms with it and manage it appropriately. </p>
+<p> As J. R. R. Tolkien says, “All we have to decide is what to do with the time that is given to us.” </p>
+
+<h3> More resources if you’re interested in reading more about this! </h3>
+
+<ul>
+    <li> <a href="http://runrunlive.com/time-scarcity-versus-time-abundance" target="_blank" rel="noopener noreferrer"><Time Scarcity versus Time Abundance></a></li>
+    <li> <a href="hhttps://www.highlights.lornerubis.com/2018/06/the-daily-fight-between-abundance-and-scarcity/" target="_blank" rel="noopener noreferrer"><The Daily Fight Between Abundance and Scarcity></a></li>
```

Comment:
my template is probably a poor example but it should be:

```
<li> <a href="http://runrunlive.com/time-scarcity-versus-time-abundance" target="_blank" rel="noopener noreferrer">Time Scarcity versus Time Abundance</a></li>
    <li> <a href="hhttps://www.highlights.lornerubis.com/2018/06/the-daily-fight-between-abundance-and-scarcity/" target="_blank" rel="noopener noreferrer">The Daily Fight Between Abundance and Scarcity</a></li>

```
    
anything in tags will need a closing tag so we don't ever use <> in html

Reply:
yeah i was a bit confused bc in your example u put <> but sounds good